debian/libmariadb3.install

```diff
@@ -1,6 +1,6 @@
 usr/lib/*/libmariadb.so.*
 usr/lib/mysql/plugin/client_ed25519.so
+usr/lib/mysql/plugin/client_ed25519.so
```

Comment:
I think this line should be removed entirely, no? Line 2 is the same. Please confirm, then I will merge the patches.

Reply:
Commit amended. Feel free to merge when you want.